.github/workflows/pycafe-dashboards-in-CI.yml

```
@@ -0,0 +1,47 @@
+name: PyCafe Dashboards in CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHON_VERSION: "3.12"
+
+jobs:
+  create-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r tools/tools_requirements.txt
+          pip install hatch
+      - name: "Build vizro-core package"
+        run: |
+          cd vizro-core
+          hatch build
```

```suggestion
          hatch build -t wheel
```
Since we only care about the wheel file.